blog/_src/posts/2017-02-15-hopl2017-linear-types.md

```diff
@@ -0,0 +1,32 @@
+    Title: Linear Types for Low-level Languages
+    Date: 2017-02-15T01:22:12
+    Tags: HOPL, by Dan Patterson
+
+TODO
+
+<!-- more -->
+
+TODO
+
+Unofficial notes: [https://github.com/nuprl/hopl-s2017/blob/master/lecture_notes/2017-02-14.md]()
+
+Just after the talk, over lunch, we had a lab discussion about the phrase
```

Comment:
I thought this was a fun discussion, but the prose here is very raw.

Up to you if you even want to keep this section --- we can also put it at the end of the `lecture_notes/`.